src/components/PickerDay.vue

```diff
@@ -413,7 +413,8 @@ export default {
      * @param {Date} date
      */
     highlightOnMouseover (date) {
-      if (this.isDisabledDate(date) || !this.highlightDate) return;
+      if(date.isDisabled || !this.highlightDate) return;
```

Comment:
`highlightOnMouseover` is called on emit `mouseover` from DaysGrid.vue where dates are not of type JavaScript Date but _date_ object built using `getDateObject` method. Therefore, we need to check date.isDisabled property - this required updating tests where `highlightOnMouseover` method was called directly without the _date_ object or the date was passed as JavaScript Date object which threw errors